test/multi_region_aws_camunda_test.go

```diff
@@ -26,7 +26,8 @@ const (
 )
 
 var (
-	remoteChartVersion = helpers.GetEnv("HELM_CHART_VERSION", "8.3.11")
+	// renovate: datasource=helm depName=camunda-platform registryUrl=https://helm.camunda.io
```

Comment:
No it would be the same PR.
Only thing that could happen is if they push a new Major and Minor at the same time, that would result in two RPs.
By default major and minor are separated.
If a new major is pushed, it would update this version here and in the `export_environment_prerequisites` as one PR.
If a new minor / patch is pushed for the existing version it would create a PR for that as well.

Example:
`10.0.2` is currently defined.

Pushed: `11.0.0` and `10.0.3`
--> 2 PRs, one with the adjustments for `11.0.0` and one for `10.0.3` but it would update those in both files at the same time, so you don't end up with 4 PRs.